bundles/specmate-integration-test/resources/test_rules.objectif

```diff
@@ -1,4 +1,4 @@
-WENN A=5 UND B=3 DANN C=7 ENDE-WENN
+WENN A=5 UND X=9 DANN C=7 ENDE-WENN
```

Comment:
The model generation does not work when special chars (e.g. [(*<)) or umlauts (ÄÖÜ etc.) are in the text
The model generation does not work when line breaks are in the text (see comment below)

Pelase fix and add tests